### PR TITLE
The syndie cola PR.

### PR DESCRIPTION
**Pull request by lordsaladin**
Hello I decided to actually code something for dm;ca.
this adds syndie cola, the end result of combining paracetamol,regenerative jelly(using tricord's code without the OD as we don't  have xenobio yet) and cherry cola.
coder sprite: 
syndie cola coder sprite
lore stuff: Donksoft's R&D where running low on funds and just had a shipment of stolen space colas, cherries .tricord , slime cores and paracetamol delivered, and they decided to reverse engineer the space colas and combine them with slime jelly, paracetamol,cherry juice and tricord to make a health drink they could sell to all sides.

### CODE REVIEW

**Comment by lordsaladin** [comment]
essentially you can create fallout style stimpacks out of this.

**Comment by Rohesie** [comment]
Looks pretty strong. What's the tradeoff?

**Comment by lordsaladin** [comment]
rarity either rng spawn in  the actual map's eateries or via tg botany's strange seeds if that gets ported.

File: code/modules/reagents/chemistry_reagents/drink.dm [hunk 1]
```diff
@@ -360,6 +360,27 @@
...
+		
+/datum/reagent/drink/cold/Syndie_cola
+	name = "Syndie cola"
+	id = "syndie_cola"
+	description = "Donksoft's first foray in to soft drinks,combining a top secrect blend of ingredients and xenobiology."
+	color = "#df182b" // rgb: 223, 24, 43
+	adj_sleepy = -4
+	custom_metabolism = 0.025 // Lasts 10 minutes for 15 units
+	
+	on_mob_life(mob/living/M, alien)
```
Comment (Rohesie) [inline code/modules/reagents/chemistry_reagents/drink.dm]: Full relative path, please.

File: code/modules/reagents/chemistry_reagents/drink.dm [hunk 2]
```diff
@@ -360,6 +360,27 @@
...
+		. = ..()
+		if(!.) return
```
Comment (Rohesie) [inline code/modules/reagents/chemistry_reagents/drink.dm]: Control statements on their own line.

**Comment by Rohesie** [comment]
Can't say I see much purpose for this, to be quite fair.

**Comment by Rohesie** [comment]
This doesn't seem to have much purpose. Feel free to port existing drinks with recipes and sprites from other codebases if you want.

**Pull request closed without merging**